spelling_wordlist.txt

```diff
@@ -112,11 +113,13 @@ precompiled
 preprocess
 preprocessing
 prestate
+prover
```

Comment:
Prover should always be capitalized, which should not be flagged as a spelling error.

Reply:
yeah, we already had `Prover` .. removing `prover`